elytron/src/main/java/org/wildfly/extension/elytron/AuditResourceDefinitions.java

```diff
@@ -106,6 +107,12 @@ class AuditResourceDefinitions {
             .setRestartAllServices()
             .build();
 
+    static final SimpleAttributeDefinition ENCODING = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.ENCODING, ModelType.STRING, true)
```

Comment:
The default value should be set or documented in the descriptions of related attributes.

Reply:
Added description that the default is UTF_8.